frontend/src/scenes/funnels/funnelLogic.ts

```diff
@@ -639,6 +657,9 @@ export const funnelLogic = kea<funnelLogicType>({
         setBinCount: async () => {
             actions.loadResults()
         },
+        setConversionWindow: async () => {
+            actions.loadResults()
```

Comment:
this is already breakpointed in `loadResults`, right?

Reply:
yupp, but i remove this whole listener in my newest changes

Reply:
sorry! just pushed them